_posts/2019-11-13-cockpit-207.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 207
+author: kkoukiou
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-207
+category: release
+summary: Cockpit with authentication and storage page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Cockpit's web server now accepts EC certificates
+
+To specify the [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) certificate
+the web service should use, simply drop a file with the extension .cert in the `/etc/cockpit/ws-certs.d` directory.
```

Comment:
OK, this is kind of klunky due to the sentence construction. It's not wrong, but it's hard to parse.

Drop has multiple meanings. The cert has ECC, but that's happenstance for putting a cert in that directory, right?

What about something like this?

```suggestion
Use a certificate with [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) by copying the certificate file into the `/etc/cockpit/ws-certs.d` directory. Make sure the certificate file has the `.cert` extension.
```

Please feel free to correct or riff off what I typed above. :wink:

(This suggestion won't merge cleanly, as the paragraph is split into two lines instead of relying on editor wrapping. This is one of the reasons we want to use auto-wrap for Markdown files in the repo.)

Reply:
@martinpitt  has posted another suggestion, I will use his since he wrote the feature better than all since he wrote that.